designs/Commands/clear-package.md

```diff
@@ -0,0 +1,92 @@
+# Clearing an specific package from NuGet cache
```

Comment:
It's not a cache, it's an installation directory. There's no updating, stale checks etc.

It's like program files.

Please refer to it as `global packages folder`.